### DISCUSSION TITLE
Checking camera status/remote downloads

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#937] [Q&A]
I have a Hero 8 that is in a racecar and setup to run automatically. It turns on with USB power, and through a script starts at speed. It's in a location where I can't see the rear screen but can get to the front of it and side buttons. Is there any way for me to verify that the camera is ready to start on speed remotely? I can tell when looking at the screen but don't have the option to do that with this one.

Another issue is that once it is setup I want to be able to download the files over USB. It seems that if I connect to a PC over USB, or even connect with the quik app on ios, it cancels my start on speed script. Is that expected? Should I power the camera down and restart it after connecting to it over USB/App?

**Comment by dnewman-gpsw** [comment]
No remote verification.  All to be expected.
